src/containers/FilmFrontpage/FilmFrontpage.jsx

```diff
@@ -202,15 +136,16 @@ FilmFrontpage.propTypes = {
   aboutNDLAVideo: PropTypes.shape({
     title: PropTypes.string,
     description: PropTypes.string,
-    langugae: PropTypes.string,
+    language: PropTypes.string,
     visualElement: PropTypes.shape({
       type: PropTypes.string,
       url: PropTypes.string,
       alt: PropTypes.string,
     }),
   }).isRequired,
-  language: PropTypes.oneOf(['nb', 'nn', 'en']).isRequired,
+  locale: PropTypes.oneOf(['nb', 'nn', 'en']).isRequired,
```

Comment:
Kanskje flytte til constants eller ligende? Ser det er brukt flere steder. Fint å slippe mange steder i fremtiden hvis vi får flere språk, for eksempel samisk :)